Kickstarter-iOS/Library/UIImageView+URL.swift

```diff
@@ -15,3 +18,30 @@ extension UIImageView {
                      completion: nil)
   }
 }
+
+private enum Associations {
+  fileprivate static var imageUrl = 0
+}
+
+extension Rac where Object: UIImageView {
```

Comment:
I don't know why we never did this before, but this is a reactive extension so that we can just do 

```swift
self.imageView.rac.imageUrl = self.viewModel.outputs.imageUrl
```

without all the cancel/set to nil/load image hoopla we've done all over the place.